lib/index.js

```diff
@@ -32,30 +32,32 @@ Floodlight.ensure(function(msg){
  *
  * https://segment.com/docs/spec/track/
  *
- * @param {Track} track
- * @param {Object} settings
+ * @param {Object} ret
  * @param {Function} fn
  * @api private
  */
 
-Floodlight.prototype.track = function(payload, done){
+Floodlight.prototype.track = function(ret, done){
   // If the payload is empty, return invalid
-  if (is.empty(payload)) return this.invalid('Only mapped events will be sent to DoubleClick Floodlight.');
-  // FL wants you to send the parameters as an endpoint delimited by semi-colons rather than get query params
-  var endpoint = [
-   'dc_rdid=' + payload.dc_rdid,
-   'src=' + payload.src,
-   'cat=' + payload.cat,
-   'type=' + payload.type,
-   'ord=' + payload.ord,
-   'tag_for_child_directed_treatment=' + payload.tag_for_child_directed_treatment,
-   'dc_lat=' + payload.dc_lat
-  ];
-
-  endpoint = endpoint.join(';');
+  if (!ret.endpoints.length) return;
```

Comment:
it used to return an error message but that's probably to noisy since that would happen for every `.track()` event that hasn't been mapped in your setting. What is the proper way to noop without hanging? Just `done();`? @tsholmes